### PR TITLE
consolidate to one CDK crate

### PR DESCRIPTION
**Pull request by thesimplekid**
Having the two crates adds complexity without benefit since features can be used instead.

Credit: Thought about this after seeing Yuki's fork

### CODE REVIEW

**Comment by thesimplekid** [comment]
@yukibtc  @lorenzolfm @davidcaseria thoughts?

**Comment by lorenzolfm** [comment]
I agree with this approach. Seems much simpler :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,14 +26,12 @@ jobs:
...
-            -p cashu --no-default-features --features mint,
-            -p cashu --no-default-features --features all-nuts,
-            -p cashu-sdk,
-            -p cashu-sdk --no-default-features,
-            -p cashu-sdk --no-default-features --features all-nuts,
+            -p cdk,
+            -p cdk --no-default-features,
+            -p cdk --no-default-features --features wallet,
+            -p cdk --no-default-features --features mint --features redb,
+            -p cdk --no-default-features --features mint --features redb,
```
Comment (yukibtc) [inline .github/workflows/ci.yml:33]: This line should be removed

File: misc/scripts/check-crates.sh [hunk 2]
```diff
@@ -24,11 +24,10 @@ fi
 
 buildargs=(
     "-p cashu"
```
Comment (yukibtc) [inline misc/scripts/check-crates.sh:26]: This should be renamed to `cdk`

**Review by yukibtc — CHANGES_REQUESTED** [review]
Looks good. I found only 2 things to fix

**Pull request merged**